federal/footer.js

```diff
@@ -0,0 +1,27 @@
+const blockConfig = {
```

Comment:
This is an interesting case. The federal project was meant for strictly _content_ related use-cases.

I would therefor think this should live within the global navigation / footer code in milo, while the authoring should/could happen on the federal project such as `federal/globalnav/project-x/nav-1`